logging-modules/logback/pom.xml

```diff
@@ -50,6 +50,17 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>javax.mail</groupId>
+            <artifactId>mail</artifactId>
+            <version>1.4.7</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.activation</groupId>
+            <artifactId>activation</artifactId>
+            <version>1.1.1</version>
```

Comment:
Let's define the version numbers in properties rather than inline.

Reply:
I had done that initially but later removed those as part of copying dependencies to the post (forgot to redo them).
Done now, please check,